### PR TITLE
Config/update toml

### PR DESCRIPTION
**Pull request by tvpeter**
# ↪️ Pull Request

Add wallet data for testing and specify Rustc minimum version

## 💻 What does this PR do?

- [x] Specify Rustc minimum version
- [x] Add wallet data for testing
- [x] Update `docs` to reflect how to load and run tests 

## ✔️ Check List
- [x] Tests has been run in packages where changes made if available
- [x] Linter has been run for changed code
- [x] Tests for the changes have been added if not covered yet
- [x] Docs have been added / updated if required
- [x] Filled out test instructions (In case there aren't any unit tests)

## 🔒 Related Issues

- Closes #

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,21 +67,23 @@ Read more here [sqlx-cli](https://crates.io/crates/sqlx-cli)
...
-
-7. To get unspent transaction outputs (UTXO), use the command `bitcoin-cli listunspent`
-
-8. Replace all addresses, public keys and txid's in the appropriate tests
-
-9. TO get the redeemscript, run the test in the funding transaction, and it will output the redeemscript, then replace the stated redeemscript in the test_redeemscript fn to run your test.
-
-10. Run `cargo test` to run all the tests
+2. (a) To make it easier for you to test, we have included `test` wallet data. To make use of the data in your Bitcoin node, use the following command to load the data and wallet:
+    -   `bitcoind -datadir=$(pwd)/.bitcoin` # start bitcoind
```
Comment (Jeezman) [inline README.md:71]: @tvpeter Will this override the existing wallet data?

File: README.md [hunk 2]
```diff
@@ -67,21 +67,23 @@ Read more here [sqlx-cli](https://crates.io/crates/sqlx-cli)
...
+
+  (b) If you prefer to use your existing wallet, follow the below steps:
+    -   Generate a new address using `bitcoin-cli getnewaddress` command.
+    -   Kindly ensure that you are using unique addresses from different wallets 
+    -   To switch from one wallet to another, use `bitcoin-cli loadwallet {walletname}` 
+    -   To generate an address from a wallet, specify the wallet e.g `bitcoin-cli -rpcwallet={walletname} getnewaddress`
+    -   To get a public key, use an already generated address and use the cli command `bitcoin-cli getadressinfo {address}` 
+    -   To get unspent transaction outputs (UTXO), use the command `bitcoin-cli listunspent`
+    -   Replace all addresses, public keys and txid's in the appropriate tests 
+    -   TO get the redeemscript, run the test in the funding transaction, and it will output the redeemscript, then replace the stated redeemscript in the test_redeemscript fn to run your test.
```
Comment (Jeezman) [inline README.md:84]: Good to be explicit here too

**Review by Jeezman — COMMENTED** [review]
Great job @tvpeter. Left a few comments

File: README.md (hunk 1 above)
Comment (tvpeter) [reply README.md:71]: no it will not. This is just dummy data that contains the public keys and addresses that were used in tests

File: README.md [hunk 3]
```diff
@@ -67,21 +67,23 @@ Read more here [sqlx-cli](https://crates.io/crates/sqlx-cli)
...
+    Goto step (3) to run the tests.
+
+  (b) If you prefer to use your existing wallet, follow the below steps:
+    -   Generate a new address using `bitcoin-cli getnewaddress` command.
+    -   Kindly ensure that you are using unique addresses from different wallets 
+    -   To switch from one wallet to another, use `bitcoin-cli loadwallet {walletname}` 
+    -   To generate an address from a wallet, specify the wallet e.g `bitcoin-cli -rpcwallet={walletname} getnewaddress`
+    -   To get a public key, use an already generated address and use the cli command `bitcoin-cli getadressinfo {address}` 
+    -   To get unspent transaction outputs (UTXO), use the command `bitcoin-cli listunspent`
+    -   Replace all addresses, public keys and txid's in the appropriate tests
```
[the remark this answers is not in the document]
Comment (tvpeter) [reply README.md:83]: So I didn't list them because the tests cases using either pubkeys or addresses might increase as we continue adding. However, if they are within the sample wallet, they will work.

[the remark this answers is not in the document]
Comment (tvpeter) [reply README.md:83]: Besides, those using the first option won't need to replace anything. The second option is only applicable for those who want to bootstrap the entire configuration.

[the remark this answers is not in the document]
Comment (Jeezman) [reply README.md:83]: I think my concern is how will someone new know which ones are the tests?

File: README.md (hunk 2 above)
Comment (tvpeter) [reply README.md:84]: the ideal situation is that a user needs to create a 2-3 multisig redeemscript using their Bitcoin node's RPC commands with the same pubkeys used in the previous test, but not all users can do that.  That's why I asked to them to do, but we might even need to remove the entire documentation for the second approach as it is cumbersome.

File: README.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (tvpeter) [reply README.md:83]: Is it not better we remove the second approach and only leave the first? because that will eliminate any confusion

[the remark this answers is not in the document]
Comment (Jeezman) [reply README.md:83]: I think both are still okay

[the remark this answers is not in the document]
Comment (Jeezman) [reply README.md:83]: I think you can add some comments in the code where the tests are

[the remark this answers is not in the document]
Comment (tvpeter) [reply README.md:83]: alright

[the remark this answers is not in the document]
Comment (tvpeter) [reply README.md]: done listing all the modules

**Review by Jeezman — APPROVED** [review]
LGTM!
